_posts/2022-06-21-ElixirAHM22.md

```diff
@@ -0,0 +1,52 @@
+
+On the second day, the morning of the 8th of June, Nicola Soranzo presented in the ELIXIR Communities and Connectedness session. His topic was about [“the Galaxy community: “Gateway for integration across Communities” ](https://docs.google.com/presentation/d/1Ya-UDX5QuOVKEVQ1RuKZqdful4y425vh5qyap0QWoyU/edit?usp=sharing).
+
+{% twitter https://twitter.com/BSerranoSolano/status/1534831098659741697?t=lHqDn05ThparskDhFRk4lQ&s=09 %}
+
+On the afternoon of the 8th of June, Galaxy was also present in the poster session. Anup Kumar summarised his work on [An accessible artificial intelligence infrastructure using a docker-based Jupyterlab in Galaxy](https://drive.google.com/file/d/1a96OvZpNjLEFK8UqFrNgt7SOof5NFyA6/view?usp=sharing), which showcases a few recent scientific use cases of GPU-enabled Jupyterlab, being used for creating end-to-end artificial intelligence projects running entirely on Galaxy Europe.
+
+Also in the afternoon of the same day, Beatriz Serrano-Solano participated in the [“COVID-19” workshop](https://docs.google.com/document/d/1FKBYBxehpaxFT_vLURwq5TbyQjxI4sYiSKeULT0ioso/edit), presenting the [work done by the Galaxy community](https://docs.google.com/presentation/d/1scNZjk61eCZzk3qrKos96zKcBoFFJP6BdqRZ302iT5M/edit#slide=id.g128cc83e3ba_1_4) since the beginning of the pandemic and highlighting the importance of Open Science over the last two years.
+
+
+{% twitter https://twitter.com/CaroleAnneGoble/status/1534529389697937409 %}
+{% twitter https://twitter.com/bjoerngruening/status/1534527617939939332?t=2z2d7QvvIUWp56aY0gtYBg&s=09 %}
+
+
+On the third and last day of the event, the 9th of June, several talks were given by Björn Grüning. In the morning, he presented at the [Enabling Science at European Scale session](https://docs.google.com/document/d/1uqF_AogQ2N4hLstlzH1QGsRoWE0IHKmeLMKVBkYfSb0/edit), in which he talked about the different ways how to scale Galaxy. Just before lunch, Björn and others also gave a short talk in the [Science driven improvements to tools ecosystems workshop](https://docs.google.com/document/d/1VA_9_1TOECPL26exs4Nyow_Y2V9RGzOnzF0WYcmLxyo/edit#heading=h.8yacamuak08k). In the afternoon, Björn has participated in two parallel workshops; the first was about [single cell](https://docs.google.com/document/d/13pOfy2nkajjzsjvlGV_ch_0EvB6say1LBdOKtbXyVsY/edit#) and the second was about [ELIXIR tools platform](https://docs.google.com/document/d/14XEkip1g0S8-eqySM7uZMCXFzPqjSQX-b-i1qxVwxrU/edit) where he talked about [BioConda](https://anaconda.org/bioconda) and [Biocontainers](https://biocontainers.pro/). 
```

Comment:
```suggestion
On the third and last day of the event, the 9th of June, several talks were given by Björn Grüning. In the morning, he presented at the [Enabling Science at European Scale session](https://docs.google.com/document/d/1uqF_AogQ2N4hLstlzH1QGsRoWE0IHKmeLMKVBkYfSb0/edit), in which he talked about the different ways how to scale Galaxy. Just before lunch, Björn and others also gave a short talk in the [Science driven improvements to tools ecosystems workshop](https://docs.google.com/document/d/1VA_9_1TOECPL26exs4Nyow_Y2V9RGzOnzF0WYcmLxyo/edit#heading=h.8yacamuak08k). In the afternoon, Björn participated in two parallel workshops; the first was about [single cell](https://docs.google.com/document/d/13pOfy2nkajjzsjvlGV_ch_0EvB6say1LBdOKtbXyVsY/edit#) and the second was about [ELIXIR tools platform](https://docs.google.com/document/d/14XEkip1g0S8-eqySM7uZMCXFzPqjSQX-b-i1qxVwxrU/edit) where he talked about [BioConda](https://anaconda.org/bioconda) and [Biocontainers](https://biocontainers.pro/). 
```

Reply:
I could not be at the ELIXIR tools platform as this was overlapping with the Single-Cell one. In Single-cell I talked about workflowhub.